coreneuron/network/multisend.cpp

```diff
@@ -1,8 +1,8 @@
 #include "coreneuron/nrniv/nrniv_decl.h"
-#include "coreneuron/nrniv/multisend.h"
-#include "coreneuron/nrnoc/multicore.h"
-#include "coreneuron/nrniv/netcon.h"
-#include "coreneuron/nrniv/netcvode.h"
+#include "multisend.hpp"
+#include "coreneuron/sim/multicore.hpp"
+#include "netcon.hpp"
+#include "netcvode.hpp"
```

Comment:
Just to be sure I understand include policy for consistency : if header is in current directory, we are not using full `coreneuron/` path.

Reply:
no we should be using it.. that's what we agreed on, but I did miss a few files (laziness...) I'll fix this once and for all